system-test/test-middleware-express.ts

```diff
@@ -0,0 +1,85 @@
+/*!
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* AN EXAMPLE RATHER THAN A TEST AT THIS POINT */
+
+import * as assert from 'assert';
+import * as delay from 'delay';
+
+import {express as elb} from '../src/index';
+
+const logging = require('@google-cloud/logging')();
+
+const WRITE_CONSISTENCY_DELAY_MS = 20 * 1000;
+const TEST_TIMEOUT = WRITE_CONSISTENCY_DELAY_MS + (10 * 1000);
+
+describe('express middleware', () => {
+  // mocha doesn't like async describe callbacks. So we have to initialize
+  // logger, mw in the before block. Extra gymnastics are required to properly
+  // type variables that seemingly (to the compiler) are conditionally
+  // intialized.
+  // https://github.com/mochajs/mocha/issues/2975
```

Comment:
Nothing to do here, but since `logger` and `mw` are the interfaces being tested, shouldn't they be initialized in a `before` callback block anyway?